src/wordle_solver/solver.py

```diff
@@ -4,15 +4,17 @@
 import time
 from typing import List, NewType, Union
 
-from definitions import ROOT_DIR
 from selenium import webdriver
 from selenium.webdriver.chrome.options import Options
 from selenium.webdriver.common.by import By
 from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as ec
 from selenium.webdriver.support.ui import WebDriverWait
-from selenium.webdriver.support import expected_conditions as EC
+
+from definitions import ROOT_DIR
 
 GAME_URL: str = "https://www.powerlanguage.co.uk/wordle/"
+# GAME_URL: str = "https://vue-wordle.netlify.app"
```

Comment:
Backup URL? Did you mean to leave this in?

Reply:
My bad, it's a backup link.